etc/config/solidity.defaults.properties

```diff
@@ -1,11 +1,18 @@
-compilers=/usr/bin/solc
-compilerType=solidity
+compilers=solc:zksolc
```

Comment:
Is this change going to result in any previous short links breaking?

Reply:
Could you please elaborate, why could it break any short links and how?

Reply:
I was concerned the compiler name would change which could cause past links with the old compiler name to break *but* I just realized this is `solidity.defaults.properties` rather than ``solidity.amazon.properties`, so changes here should be fine :)